docs/source/tutorial4_fhir_api.rst

```diff
@@ -0,0 +1,187 @@
+Request on FHIR API
+===================
+
+After :ref:`starting FHIR server<tutorial2_docker_run:start fhir server>` and
+:ref:`adding data to GameBus platform<tutorial3_gamebus:add data manually>` in
+previous sections, now it's ready to try the service of GameBus FHIR layer.
+
+The FHIR layer is a service layer on GameBus platform, providing FHIR REST API
+for communicating FHIR-compliant data with outside. For example, when a user sends
+an HTTP GET request to FHIR REST API, the FHIR layer will transform this request and
+forward the transformed request to GameBus's API; GameBus will process the request,
+e.g. reading requested data from its database, and then return the relevant data in
+GameBus format to the FHIR layer; the FHIR layer will then transform the GameBus data to
+data in FHIR-compliant format, and then return the FHIR data to user as a HTTP
+response. That is how FHIR layer works and how it enables GameBus to provide FHIR
+service.
+
+To try and test FHIR API, we need API clients to help send HTTP requests.
+Various API clients exist, e.g. `httpie`_, `Postman`_ and `Hoppscotch`_.
```

Comment:
I'd say curl can also be an option right?

Reply:
certainly